docs/plugins/bgp.policy.md

```diff
@@ -16,17 +16,21 @@ The **bgp.policy** plugin implements simple BGP routing policies :
 
 The plugin adds the following BGP attributes:
 
+* **bgp.bandwidth** link attribute sets the BGP Link Bandwidth extended community. It can be an integer (in megabytes), in which case the Link Bandwidth community is attached to inbound EBGP updates[^BCP], or a dictionary with **in** and **out** integer values if you want to set the Link Bandwidth community in both directions (or just on the outbound updates).
```

Comment:
My use case was actually having the router append the bandwidth value corresponding to the interface it was received on, such that it can do UCMP on the routes

In a sample deployment, a customer would have 100G links and then add a new 400G link. Rather than getting ECMP with at most 100G over the new link, we'd want to send 4x the amount across the new path

Reply:
Got it. That's how IOS does things. OTOH, some other devices can't even change the bandwidth on an interface :(

Anyhow, we could make the attribute take an integer value (where supported) or **auto** for "use interface bandwidth", like **bgp.bandwidth: auto** or **bgp.bandwidth.in: auto**

If that works for you I'll fix the parser and the check for device support.